docs/developers_guide/Testing.md

```diff
@@ -35,27 +35,42 @@ val scenario =
 ```
 
 ### Creating test scenario runner
-Scenario should be executed inside a runner. `NuTestScenarioRunner` gives you another DLS for building runners.
+
+Scenario should be executed inside a runner. `TestScenarioRunner` gives you another DLS for building runners.
 At first, you chose type of the scenario from:
-- flink - only one right now
+- flinkBased - based on Flink engine, you need to pass to it `FlinkMiniClusterHolder`, it can be created e.g. using `FlinkSpec`:
 
 ```scala
- val testScenarioRunner = NuTestScenarioRunner
-      .flinkBased(config, flinkMiniCluster)
-      .build()
+import pl.touk.nussknacker.engine.flink.util.test.FlinkTestScenarioRunner._
+val testScenarioRunner = TestScenarioRunner
+  .flinkBased(config, flinkMiniCluster)
+  .build()
 ```
 
-:::caution
-For flink runner test must extend FlinkSpec for now.
-:::
+- liteBased - based on Lite engine, no other setup needed, provides interface to communicate with engine using raw classes (similar interface as using flinkBased):
```

Comment:
```suggestion
- liteBased - based on Lite engine, no other setup needed, provides interface to communicate with engine using raw classes (not using any Kafka API, similar interface as using flinkBased):
```

Reply:
I changed. The order of test runners descriptions were changed after this.